packages/flutter_tools/lib/src/migrations/cmake_custom_command_migration.dart

```diff
@@ -56,9 +56,20 @@ class CmakeCustomCommandMigration extends ProjectMigrator {
         final String addCustomCommandReplacement = '$addCustomCommandOriginal\n  VERBATIM';
         newProjectContents = newProjectContents.replaceAll(addCustomCommandOriginal, addCustomCommandReplacement);
       }
+
+      // CMake's add_custom_command() should add FLUTTER_TARGET_PLATFORM to support multi-architecture.
+      // However, developers would get the following warning every time if we do nothing.
+      // ------------------------------
+      // CMake Warning:
+      //   Manually-specified variables were not used by the project:
+      //    FLUTTER_TARGET_PLATFORM
+      // ------------------------------
+      if (addCustomCommandOriginal?.contains('linux-x64') == true) {
```

Comment:
Is `== true` intentional for some reason? I cannot come up with one which prevents to just write `if (addCustomCommandOriginal?.contains('linux-x64'))`

Reply:
Yes, it avoids a warning. Could you check the below.
![image](https://user-images.githubusercontent.com/62131389/106423639-2d28e100-64a4-11eb-82b9-f5d5cb6ba05f.png)

Reply:
`?.contain` may result in `null`

Reply:
That's right.

Reply:
Shall `?? false` do the trick or is it not yet in the stable channel and that's the reason?

Reply:
I would prefer `==true` for condition while `?? false` for assignment